### PR TITLE
Invoice email design

### PR DESCRIPTION
**Pull request by Shruti-Apte**
## Notion card
https://www.notion.so/saeloun/Clients-should-be-able-to-view-invoices-from-email-Frontend-01b4088de4cf4f3bac5524802a65782d
## Summary

Designed Invoice Email page

## Preview

emailInvoice
## Type of change

Please delete options that are not relevant.
- [ ] New feature (non-breaking change which adds functionality)

### Checklist:

- [ ] I have manually tested all workflows
- [ ] I have performed a self-review of my own code

### CODE REVIEW

File: app/views/layouts/mailer.html.erb [hunk 1]
```diff
@@ -3,7 +3,108 @@
...
+        font-weight:normal;
+      }
+
+      .copyright_label {
+        font-size:12px;
+        font-weight:normal;
+        line-height: 16px;
+        text-align: center;
+      }
+
```
Comment (akhilgkrishnan) [inline app/views/layouts/mailer.html.erb:107]: Are we planning to use the same design for all the emails from Miru. If not please remove the uncommon styles to invoice.html.erb itself

Comment (Shruti-Apte) [reply app/views/layouts/mailer.html.erb:107]: Not sure if we are using this for all email designs

Comment (shivamsinghchahar) [reply app/views/layouts/mailer.html.erb:107]: This layout file should contain only the common styles which can be used by other mailers e.g. color classes, container classes, etc.,

The specific styles like amount, details_title, etc., belong to the `../invoice_mailer/invoice.html.erb` file.

**Comment by apoorv1316** [comment]
@Shruti-Apte Few concerns
1. Any idea why I can't see Saeloun and Miru logo in my local. You can see there is no logo in the image.
2. Also, why the currency symbol is not being displayed with the amount.
3. On the view invoice page the footer is not fixed to the bottom and it looks odd at my screen resolution.
Screenshot 2022-05-27 at 10 17 56 AM
Screenshot 2022-05-27 at 10 26 49 AM

**Comment by Shruti-Apte** [comment]
> Why are there so many commits and file changes? You may want to cherry-pick your changes or raise the PR on another base branch

Yes Actually my laptop was crashing continuously ..was facing issues like "bad object head" with git so used few commands to fix it like  " git reset --hard head" and had to rebase it with develop ! That may have been caused this issue according to me!

I will raise a PR on another base branch, and will make sure this doesn't happen next time !

**Review by supriya3105 — CHANGES_REQUESTED** [review]
Change the date format to dd-mm-yyyy at all the places
Logo and company name should be dynamic on invoice email
Remove print and download from client invoice view
Remove pay via paypal
The stripe payment link is not working.

**Comment by supriya3105** [comment]
@apoorv1316 Please test your review comments. I have tested it on Shruti's local and except for the currency symbol everything is working. Added the currency symbol to another Notion card.

**Comment by apoorv1316** [comment]
@supriya3105 We can merge it then. I'll approve it.

**Review by apoorv1316 — APPROVED** [review]

**Comment by supriya3105** [comment]
Closing this as new PR has been raised.

**Pull request closed without merging**